airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/retriers/chain_retrier.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from typing import List, Union
+
+import requests
+from airbyte_cdk.sources.declarative.requesters.retriers.retrier import (
+    NonRetriableResponseStatus,
+    ResponseStatus,
+    Retrier,
+    RetryResponseStatus,
+)
+
+
+class ChainRetrier(Retrier):
```

Comment:
nit: just a naming thing, the name "retrier" threw me off a bit because it doesn't convey that we can ignore errors. Is error handler a better naming direction to go in? or response handler? can you think of any better names? I'm drawing a blank :D